pkg/test/instrumentation.go

```diff
@@ -0,0 +1,64 @@
+package test
```

Comment:
Keep this in environment.go for now, we might eventually need to split it into its own package but everything else is in that file.

Reply:
done